docs/release-notes/1.9.0.md

```diff
@@ -0,0 +1,5 @@
+## New Features
+
+- Pods with proxy sidecars will be restarted in chunks on Istio upgrade in multiple reconciliations and not all at once. This will increase stability and reliability of the reconciliation for the Istio module operator. [issue #155](https://github.com/kyma-project/istio/issues/155)
```

Comment:
```suggestion
- During the Istio upgrade, Pods with the Istio sidecar proxies will now be divided into smaller groups and restarted in multiple reconciliations instead of all at once. This increases the stability and reliability of the reconciliation for the Istio module's operator. See the [issue #155](https://github.com/kyma-project/istio/issues/155).
```